src/Database/PostgreSQL/Simple/ToRow.hs

```diff
@@ -198,6 +200,9 @@ instance (ToField a, ToField b, ToField c, ToField d, ToField e, ToField f,
 instance (ToField a) => ToRow [a] where
     toRow = map toField
 
+instance (ToField a) => ToRow (NonEmpty a) where
```

Comment:
Please add corresponding `FromRow` instance as well.

Reply:
`FromRow` is out of the scope of this issue. I don't mind if one would implement this as the separate issue/PR.

Reply:
I'm sorry, but I won't accept a PR without symmetry.

Reply:
Then you should have told that in the issue (which clearly states what is needed to be implemented) and you don't waste your time.